LEAF_Request_Portal/templates/initial_form.tpl

```diff
@@ -86,8 +86,8 @@ $(function() {
             <tr>
                 <td>Priority</td>
                 <td>
-                    <select id="priority" name="priority" aria-label="priority">
-                    <option value="-10">EMERGENCY</option>
+                    <select id="priority" name="priority" aria-label="priority" style="width: 150px;">
+                    <option value="10">EMERGENCY</option>
```

Comment:
The value of the option was changed from "-10" to "10", it should remain "-10"

Reply:
Sorry. Changed it back to -10.